iModelCore/GeomLibs/geom/test/PolyfaceTest/t_ClipPlaneSet.cpp

```diff
@@ -2666,3 +2666,43 @@ TEST(ClipPlaneSet,CloneBetweenDirectedFractions)
 
     Check::ClearGeometry("ClipPlaneSet.CloneBetweenDirectedFractions");
     }
+
+TEST(ClipPlaneSet, ClipParityRegionSweep)   // verifies the fix for Bug #1279805

```

Comment:
Not sure if merging this comment makes sense. All code changes are to fix some bugs!

Reply:
This is an old habit I found immensely useful in pre-GitHub days, but you're right. With VSCode's GitHub integration giving context for every line of code, it's mostly not needed anymore.